editor/editor_inspector.cpp

```diff
@@ -1021,7 +1021,7 @@ void EditorInspectorSection::setup(const String &p_section, const String &p_labe
 			vbox->hide();
 		}
 	}
-		//	void editor_set_section_unfold(const String &p_section, bool p_unfolded);
+	//	void editor_set_section_unfold(const String &p_section, bool p_unfolded);
```

Comment:
srsly.. this is literally what my local clang made me do ... now travis hates it